scripts/assertStateEquals.js

```diff
@@ -0,0 +1,27 @@
+const SnappBase = artifacts.require("SnappBase")
```

Comment:
Maybe assert is the wrong name, as we do have a real assert. 

Throwing an error will probably also kill the whole testing script, which is bad.

I will keep on investiagting.